animation_nodes/nodes/mesh/get_vertex_group.py

```diff
@@ -0,0 +1,26 @@
+import bpy
+from ... data_structures import DoubleList
+from ... base_types import AnimationNode
+
+class GetVertexGroupNode(bpy.types.Node, AnimationNode):
+    bl_idname = "an_GetVertexGroupNode"
+    bl_label = "Get Vertex Group"
+    errorHandlingType = "EXCEPTION"
+
+    def create(self):
+        self.newInput("Mesh", "Mesh", "mesh")
+        self.newInput("Text", "Name", "groupName")
+        self.newOutput("Float List", "Weights", "weights")
+
+    def execute(self, mesh, attributeName):
```

Comment:
Don't use the generic attribute name, use group instead. `attributeName -> vertexWeightName`, `Vertex group name can't be empty. -> Group name can't be empty.`, `...`.

Reply:
Omar, we also have *Face Groups*, so we should use the full name *Vertex Group*, isn't it?

Reply:
Yes, my comment above was typo, make it vertex group.